frontend/src/styles/color-variables.sass

```diff
@@ -19,6 +19,7 @@ $dark-grey: #121415
 $charcoal-grey: #222327
 $deep-dark-grey: #1D1E21
 $border-grey: #919193
+$scrollbar-color: rgba(255, 255, 255, 0.25)
```

Comment:
it would be better to use the color name instead of the item name where we apply this color